azure-pipelines.yml

```diff
@@ -10,7 +10,7 @@ trigger:
 strategy:
   matrix:
     linux:
-      imageName: 'ubuntu-latest'
+      imageName: 'ubuntu-18.04'
```

Comment:
Mongodb url can't be resolved for the latest version of ubuntu, we should temporarily use ubuntu-18.04 until https://github.com/mongodb-js/download-url/ will be updated to fix the problem.